website/docs/reference/integrations/jira-cloud-plugin-usage.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Jira Cloud Integration - Usage
+---
+
+With the Unleash Jira Cloud Plugin you can create, view and manage Unleash feature toggles directly from a Jira issue.
+
+The plugin also shows you current status of connected toggles.
+
+## Using the plugin
+
+To use the plugin, you'll need to create an issue or use an existing one. Once the issue is saved and you open up the issue panel, you'll be greeted with a button to activate the Unleash plugin for that issue.
+
+![Jira Cloud - Button to activate Unleash plugin in Jira Issue](/img/jira-cloud-activate-unleash-panel-button.png)
+
+### Connecting a toggle to an issue
+
+![Jira Cloud - Button that opens modal to connect toggles to Jira Issue](/img/jira-cloud-issue-button.png)
+
+Clicking this button opens a dialog where you first select an Unleash Project to connect toggle from,
+
+![Jira Cloud - Dropdown to select Unleash project](/img/jira-cloud-select-project-expanded.png)
+
+allowing you then to either connect an existing toggle
+
+![Jira Cloud - Dropdown to select existing toggle to connect](/img/jira-cloud-add-existing-toggle.png)
+
+or if you flip the switch that says "Create new toggle", allows you to create a new toggle
+
+![Jira Cloud - Switch that opens the form to create and connect a new toggle](/img/jira-cloud-add-new-toggle.png)
```

Comment:
You might need to jimmy the indentation around here to make the list render properly.

```suggestion
2. Use the "create new toggle" option to choose whether you want to add an existing toggle or create a new one.  

<Figure caption="When you add an existing toggle, use the toggle name select list to choose from existing toggles in the selected Unleash project." img="/img/jira-cloud-add-existing-toggle.png" />

<Figure caption="When you add a new toggle, you must give it a name and can choose to give it a description." img="/img/jira-cloud-add-new-toggle.png" />c
```

Reply:
Yup, I'll accept the changes and inspect the rendering after